<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>606</width>
        <height>977</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="config_name_label">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Config Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="config_name_line_edit">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="autopen_group_box">
         <property name="title">
          <string>AutoOpen</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>DelayTime(ms)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Switchboard</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Current Machine IP Address. Used as OSC send target</string>
              </property>
              <property name="text">
               <string>IP Address</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="ip_address_line_edit">
              <property name="toolTip">
               <string>Current Machine IP Address. Used as OSC send target</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="transport_path_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Transport Path</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="transport_path_line_edit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>Feature coming soon</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="listener_exe_label">
              <property name="toolTip">
               <string>Filename of the Switchboard listener executable. (The .exe extension can be omitted on Windows.)</string>
              </property>
              <property name="text">
               <string>Listener Executable Name</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="listener_exe_line_edit">
              <property name="toolTip">
               <string>Filename of the Switchboard listener executable. (The .exe extension can be omitted on Windows.)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Project Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="project_name_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Project Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="project_name_line_edit">
              <property name="toolTip">
               <string>Name of your project. All takes will be recorded into a folder structure with this name as the root name.</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>MyProject</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="uproject_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>UProject</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLineEdit" name="uproject_line_edit">
                <property name="toolTip">
                 <string>Path to uProject</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>C:\Users\UserName\Documents\Unreal Projects\MyProject\MyProject.uproject</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="uproject_browse_button">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="engine_dir_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Engine Dir</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLineEdit" name="engine_dir_line_edit">
                <property name="toolTip">
                 <string>Path to UE4 Engine directory</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>C:\Program Files\Epic Games\UE_4.26\Engine</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="engine_dir_browse_button">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Build Engine</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="build_engine_checkbox">
              <property name="toolTip">
               <string>Check if engine needs to be built from source</string>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="map_path_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Map Path</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="map_path_line_edit">
              <property name="toolTip">
               <string>Relative path from Content folder that contains maps to launch into.</string>
              </property>
              <property name="placeholderText">
               <string>Maps</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="map_filter_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Map Filter</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="map_filter_line_edit">
              <property name="toolTip">
               <string>Walk every file in the Map Path and run a fnmatch to filter the file names</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>*.umap</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>OSC</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QFormLayout" name="formLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Server Port</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Client Port</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="osc_server_port_line_edit"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="osc_client_port_line_edit"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="p4_group_box">
            <property name="title">
             <string>Source Control</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="p4_project_path_label">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>P4 Project Path</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="p4_project_path_line_edit">
                 <property name="placeholderText">
                  <string>//Depot/Project   (.uproject Directory)</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="source_control_workspace_label">
                 <property name="text">
                  <string>Workspace Name</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="source_control_workspace_line_edit"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="p4_engine_path_label">
                 <property name="text">
                  <string>P4 Engine Path</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="p4_engine_path_line_edit">
                 <property name="placeholderText">
                  <string>//Depot/Engine    (Engine Directory)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Multi User Server</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="mu_server_name_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Server Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="mu_server_name_line_edit">
              <property name="toolTip">
               <string>Path to uProject</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>MU_Server</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="mu_cmd_line_args_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Command Line Args</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mu_cmd_line_args_line_edit">
              <property name="toolTip">
               <string>MU Server Command line arguments</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>-UDPMESSAGING_TRANSPORT_UNICAST={IP_ADDRESS}:5555'</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Auto Join</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="mu_auto_join_check_box">
              <property name="toolTip">
               <string>Unreal Machines that have been launched join the Multi User Server</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Clean History</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="mu_clean_history_check_box">
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Auto Launch</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="mu_auto_launch_check_box">
              <property name="toolTip">
               <string>Launch the Multi User server when Unreal is launched</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="muserver_exe_label">
              <property name="toolTip">
               <string>Filename of the multi-user server executable. (The .exe extension can be omitted on Windows.)</string>
              </property>
              <property name="text">
               <string>Executable Name</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="muserver_exe_line_edit">
              <property name="toolTip">
               <string>Filename of the multi-user server executable. (The .exe extension can be omitted on Windows.)</string>
              </property>
              <property name="text">
               <string>UnrealMultiUserServer</string>
              </property>
              <property name="placeholderText">
               <string>UnrealMultiUserServer</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="muserver_auto_build_label">
              <property name="toolTip">
               <string>Whether to include the UnrealMultiUserServer project when performing local device builds.</string>
              </property>
              <property name="text">
               <string>Auto Build</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QCheckBox" name="muserver_auto_build_check_box">
              <property name="toolTip">
               <string>Whether to include the UnrealMultiUserServer project when performing local device builds.</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="device_override_layout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
